packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -39,6 +43,16 @@ import {
 } from '../forceProjectCreate';
 
 export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
+  public readonly relativeMetdataTempPath = path.join(
+    '.sfdx',
+    'isvdebugger',
+    'mdapitmp'
```

Comment:
Are these worth pulling out to constants?

Reply:
I'd say they're worth pulling out into constants if they're used in multiple places or if there is a repository of common directories/paths defined elsewhere.

Reply:
Oh, I meant isvdebugger and mdapitmp